Vic2ToHoI4Tests/Vic2WorldTests/Localisations/LocalisationsTests.cpp

```diff
@@ -0,0 +1,214 @@
+#include "Configuration.h"
+#include "V2World/Localisations/LocalisationsFactory.h"
+#include "V2World/Localisations/Vic2Localisations.h"
+#include "V2World/Mods/ModBuilder.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+
+
+TEST(Vic2World_Localisations_LocalisationsTests, MissingKeyGivesNullopt)
+{
+	const auto configuration = Configuration::Builder{}.setVic2Path("./").build();
+	const auto localisations = Vic2::Localisations::Factory{}.importLocalisations(*configuration);
+
+	ASSERT_EQ(localisations->getTextInLanguage("bad_key", "english"), std::nullopt);
```

Comment:
I think it should be `ASSERT_EQ(expected value, value)`.

Reply:
Fixed.